### DISCUSSION TITLE
addTileMatrix()  with overlapping tile ranges

### DISCUSSION THREAD

**Discussion by galib36** [GreenleafLab/ArchR#797] [Questions / Documentation]
Hello,
I was wondering whether it is possible to have tiles with overlapping windows using `addTileMatrix`? Currently the function supports with fixed tile width, but I need to have a tile with overlapping window, for e.g.
windows would be at these positions:

1-100

21-120

41-140

61-150

Thank you.

**Comment by rcorces** [comment]
Unfortunately, this is not possible. You could, of course, create such a matrix by making a tile matrix with tileSize = 20 and then doing some simple addition to create the tiles you want. This will probably use a lot of memory. But since overlapping tiles is not something that we feel makes sense for ATAC analysis, we dont have an implementation to allow for overlap. If you feel we are missing something, feel free to let us know.

**Reply by galib36** [reply]
Thank you for your reply. In one of our snATAC-seq with a cancer sample we see amplification of certain chromosome locations which is masking the Differential Accessibility. This phenomena is also mentioned in _**Sarah K. Denny et. al., Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility, Cell 2016**_. They corrected this amplification by assessing the number of background reads within a sliding window of 1 Mb with a step size of 500Kb. I am trying to implement similar approach for our cancer sample and that is why I needed an overlapping window tile. The numbers I mentioned on top are just a made up example to explain my requirement.

**Reply by rcorces** [reply]
Got it. Yes - that makes sense. We have plans to release a copy number estimation vignette at some point but have not had the bandwidth to put this together.

**Reply by galib36** [reply]
Oh that's great, thank you. Can you suggest me very briefly any workaround or any function I can try to achieve this?

**Reply by rcorces** [reply]
Unfortunately I dont have anything to provide at this time.

**Reply by galib36** [reply]
Thank you, I will wait for the vignette then.
